pyproject.toml

```diff
@@ -80,7 +80,7 @@ lint.select = ["B", "D", "E", "F", "I", "N", "S", "W", "PL", "PT", "UP", "RUF"]
 # D104: Missing docstring in public package
 # PLR2004: Magic value in comparison
 # RUF200: pyproject missing field (poetry doesn't follow the spec)
-lint.ignore = ["D104", "PLR2004", "RUF200", "S311"]
+lint.ignore = ["D104", "PLR2004", "RUF200", "S311", "PLR0915"]
```

Comment:
Could you add a comment on what this one is?

Reply:
For sure!